jekyll/_cci2/api-developers-guide.md

```diff
@@ -0,0 +1,476 @@
+
+## Authentication and Authorization
+
+The CircleCI API utilizes token-based authentication to manage access to the API server and validate that a user has permission to make API requests. Before you can make an API request, you must first add an API token and then verify that you are authenticated by the API server to make requests. The process to add an API token and have the API server authenticate you is described in the sections below.
+
+**Note** You may use the API token as the username for HTTP Basic Authentication, by passing the `-u` flag to the `curl` command.
+
+### Add an API Token
+
+To add an API token, perform the steps listed below.
+
+1.  Add an API token from your [account dashboard](https://circleci.com/account/api).
+2.  To test it, [view it in your browser](https://circleci.com/api/v2/me) or call the API using the command below.
+
+```sh
+$ curl https://circleci.com/api/v2/me
+```
+
+3.  You should see a JSON response similar to the example shown below.
+
+```json
+{
+  "user_key_fingerprint" : null,
+  "days_left_in_trial" : -238,
+  "plan" : "p16",
+  "trial_end" : "2011-12-28T22:02:15Z",
+  "basic_email_prefs" : "smart",
+  "admin" : true,
+  "login" : "someuser"
+ }
+```
+
+**Note:** All API calls are made in the same way, by making standard HTTP calls, using JSON, a content-type, and your API token. Please note that the JSON examples shown in this document are not comprehensive and may contain additional JSON response fields not shown in the example, based on user input and fields.
+
+### Get Authenticated
+
+To be authenticated by the API server, add an API token using your [account dashboard](https://circleci.com/account/api). To use the API token, add it to the `circle-token` query param:
+
+```sh
+curl "https://circleci.com/api/v1.1/me?circle-token=:token"
+```
+
+Alternatively, you can use the API token as the username for HTTP Basic Authentication, by passing the `-u` flag to the `curl` command:
+
+```sh 
+curl -u <circle-token>: "https://circleci.com/api/..."
+```
+
+**Note:** The colon ":" notifies `curl` that there is no password being passed.
+
+## API Endpoints
+
+The CircleCI v2 API, and its associated endpoints allow you to make HTTP calls to designated endpoints developed in the underlying CircleCI API architecture. These endpoints provide programmatic access to CircleCI services, including pipelines, workflows, and jobs. 
+
+Before working with the API and making calls, you should first have an understanding of the endpoints that are currently available, as detailed in the tables below:
+
+### User Endpoints
+
+Endpoint       | Description                       
+-----------|-------------------------------------------------------
+`/user/{id} ` | This endpoint enables you to return information about a specific user by making a GET request and passing the `id` parameter in the request
```

Comment:
To follow common conventions I'd add the method to the first column, like `GET /user/{id}` everywhere, instead of having it somewhere in the description.

I'd also shorten the description by a lot in general, from

```
This endpoint enables you to return information about a specific user by making a GET request and passing the `id` parameter in the request
```

to something like

```
Returns information about a specific user
```

Reply:
Fixed in all tables.